CacheSim/GenericHashTable.h

```diff
@@ -314,7 +378,7 @@ class GenericHashTable
     const size_t new_capacity = old_capacity ? old_capacity * 2 : 64;
 
     Elem** old_table = m_Table;
-    Elem** new_table = (Elem**) HeapAlloc(m_Heap, 0, sizeof(Elem*) * new_capacity);
+    Elem** new_table = (Elem**) VirtualMemoryAlloc(sizeof(Elem*) * new_capacity);
```

Comment:
I did test on windows, but I didn't profile it.  My experience with VirtualAlloc hasn't been as bad as I'd have thought in the past (I've gotten a full game to run at an "interactive" framerate with basically all allocations routing to virtualalloc for debugging). Does the windows heap actually do anything clever for allocations over, say, 2-4MB? At what point does it just end up going to VirtualAlloc anyway? Could we just pre-size the tables to something fairly large to avoid the frequent initial resizing?